From 5cf9eb2cc748907924342b32a3eefa78bb325de8 Mon Sep 17 00:00:00 2001
From: Jo Shields <directhex@apebox.org>
Date: Mon, 27 Nov 2017 11:22:52 -0500
Subject: [PATCH] Explicitly copy files from inside ExtensionsPath to xbuild
 dirs (#32)

---
 install-mono-prefix.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/install-mono-prefix.sh b/install-mono-prefix.sh
index 3952c63b2..527303a21 100755
--- a/install-mono-prefix.sh
+++ b/install-mono-prefix.sh
@@ -51,8 +51,8 @@ mv ${DESTDIR}${MSBUILD_INSTALL_BIN_DIR}/Mono.Build.Tasks.* ${DESTDIR}${XBUILD_DI
 cp -r $MSBUILD_OUT_DIR/Extensions ${DESTDIR}${MSBUILD_INSTALL_BIN_DIR}
 
 # Deploy files meant for the default $(MSBuildExtensionsPath)
-cp -r mono/ExtensionsPath/ ${DESTDIR}${XBUILD_DIR}
-cp -r mono/ExtensionsPath-ToolsVersion/ ${DESTDIR}${XBUILD_DIR}/${MSBUILD_TOOLSVERSION}
+cp -r mono/ExtensionsPath/* ${DESTDIR}${XBUILD_DIR}
+cp -r mono/ExtensionsPath-ToolsVersion/* ${DESTDIR}${XBUILD_DIR}/${MSBUILD_TOOLSVERSION}
 
 mv ${DESTDIR}${MSBUILD_INSTALL_BIN_DIR}/Microsoft.Common.props ${DESTDIR}${XBUILD_DIR}/$MSBUILD_TOOLSVERSION
 mv ${DESTDIR}${MSBUILD_INSTALL_BIN_DIR}/Microsoft.VisualStudioVersion.v* ${DESTDIR}${XBUILD_DIR}/$MSBUILD_TOOLSVERSION
