diff -urNad msbuild-15.8.old/build/Versions.props msbuild-15.8/build/Versions.props
--- msbuild-15.8.old/build/Versions.props	2018-07-31 22:43:36.000000000 -0400
+++ msbuild-15.8/build/Versions.props	2018-09-04 10:44:29.254077828 -0400
@@ -40,7 +40,7 @@
     <VSWhereVersion>2.2.7</VSWhereVersion>
 
     <GenApiVersion>2.1.0-prerelease-02404-02</GenApiVersion>
-    <GitVersioningVersion>2.1.23</GitVersioningVersion>
+    <GitVersioningVersion>2.2.13</GitVersioningVersion>
     <LargeAddressAwareVersion>1.0.3</LargeAddressAwareVersion>
   </PropertyGroup>
 
